### ISSUE TITLE
User profile modal looks messy with complete lists of challenges won and quests completed - add counters and show/hide buttons?

### ISSUE THREAD

**Issue by ghost** [HabitRPG/habitica#6933]
With the number of quests exceeding 40 and the growing number of available challenges the avatar stats and achievements in the profile modal may be becoming difficult to look through for active users.
1. ~~All the other achievements seem to use small font for descriptions~~
2. May the overall counters be added to the achievements and show/hide buttons like in https://github.com/HabitRPG/habitrpg/pull/6747 to the modal?

Vision:
![pic](http://i.snag.gy/qcTOt.jpg)

P.S. unfortunately, I'm not able to work on the solution of this issue myself, so I'll just be happy if it's discussed, approved and implemented by Aspiring Coders :smile:

**Comment by Alys** [comment]
I wouldn't be in favour of using a small font for the challenge and quest titles, since it's harder to read for people with visual difficulties (the same doesn't apply as much to the standard text for other achievements since after you've managed to read it once, you don't need to read it again). 

I really like the idea of a count of the quests and challenges won (even if no other changes from this issue are made). 

Hide/show links seem good.

Note though this issue: https://github.com/HabitRPG/habitrpg/issues/2373 **Split up "Stats & Achievements" to "Stats" and "Achievements" (or similar)**
It's been around for a while but it's still something we want to do. Sphinx, I'm not sure if your ideas here should be added to that issue, or if we'd want to do them now since that other issue might take a while to get to. What do you think?

**Comment by ghost** [comment]
@Alys 1. Thanks for mentioning it. OP edited to exclude the small font suggestion. 2. Thank you for linking to the other ticket. I like the idea of the achievements grid and I think it will look nice as well in the user profile modal. I believe this ticket maybe addressed **after** the #2373 is solved - to have it consistent.

Feel free to close this one for housekeeping reasons, but then I'll suggest reopening it after that (or maybe just add the [wait] label?)

**Comment by Alys** [comment]
@GitHubSphinx Good point, thanks! I've added the wait label, and dropped a note about this into #2373.

**Comment by ghost** [comment]
@Alys TY :ninja:

**Comment by SabreCat** [comment]
Does this still need the "wait" label? @Alys

**Comment by khipkin** [comment]
Yes, it's waiting on #2373.

**Comment by librarianmage** [comment]
What is the status on this?

**Comment by khipkin** [comment]
No status! It's up for grabs, as far as I know. :)

**Comment by khipkin** [comment]
#2373 was fixed in #7804, so this should be good to go.

**Comment by Alys** [comment]
Anything marked "on hold" and "website improvement", like this issue, is probably best left alone for now. We're part way through a project to make some improvements to the website, including changing the technology used for the site's front-end. Any "website improvement" issues are likely to be fixed as part of that project, and so any changes made for them now on the current site would be used for only a relatively short period and the programmer's time would be wasted to some extent.  Even when the programmer doesn't mind that, we still need to consider the time that admins spend reviewing and testing changes. We're a bit overloaded with PRs at the moment, so my personal feeling is that it's best to avoid work on low priority website improvements.

That does not apply to higher priority improvements for new features or to fix significant usability problems.

**Comment by librarianmage** [comment]
With the current redesign, hasn't this been mostly fixed?

**Comment by SabreCat** [comment]
There's probably still room for improvement (a long completion list still looks rather awkward), but it's much better these days.
